### DISCUSSION TITLE
Fixed postion

### DISCUSSION THREAD

**Discussion by ivanradojevic-web-dev** [tailwindlabs/tailwindcss#2789] [Help]
Fixed postion goes out of the container, very strange behaviour?!?
```
<nav class="bg-green-500 h-16 w-full fixed"></nav>
<nav class="bg-green-500 h-16 w-full"></nav>
```
![fixed](https://user-images.githubusercontent.com/59580150/99558085-a8114e00-29c3-11eb-97e5-ddcae397c9c4.JPG)

**Comment by erhankaradeniz** [comment]
Isn’t sticky what you want? Instead of fixed?
